Spec/RealtimeClientConnection.swift

```diff
@@ -1859,9 +1870,11 @@ class RealtimeClientConnection: QuickSpec {
                     let options = AblyTests.commonAppSetup()
                     options.autoConnect = false
                     options.authCallback = { tokenParams, callback in
-                        callback(getTestTokenDetails(key: options.key, capability: tokenParams.capability, ttl: tokenParams.ttl), nil)
+                        delay(0) {
```

Comment:
The `authCallback` shouldn't be blocking the execution because of the internal timers for time outs. You're right, this should be done on our side.